garden_ai/app/garden.py

```diff
@@ -177,61 +177,62 @@ def create(
     local_data.put_local_garden(garden)
 
     if verbose:
-        metadata = local_data.get_local_garden(garden.uuid)
+        metadata = json.dumps(local_data.get_local_garden_by_uuid(garden.uuid))
         rich.print_json(metadata)
     return
 
 
-# TODO: allow selecting by DOI.
 @garden_app.command(no_args_is_help=True)
 def add_pipeline(
-    garden_uuid: str = typer.Option(
+    garden_id: str = typer.Option(
         ...,
         "-g",
         "--garden",
-        prompt="Please enter a the name or DOI of a garden",
+        prompt="Please enter the UUID or DOI of a garden",
         help="The name of the garden you want to add a pipeline to",
         rich_help_panel="Required",
     ),
-    pipeline_uuid: str = typer.Option(
+    pipeline_id: str = typer.Option(
         ...,
         "-p",
         "--pipeline",
-        prompt="Please enter a the title or DOI of a pipeline",
+        prompt="Please enter a the UUID or DOI of a pipeline",
         help="The name of the pipeline you want to add",
         rich_help_panel="Required",
     ),
 ):
```

Comment:
🌈🦄 Heya, friend! 🦄🌈 The auto-generated help 📖 from the arguments is usually 🌟pretty good🌟, but what if we add a lil' 1-sentence docstring 💬 here, like "add a registered pipeline to a garden" 🌷🌻? Methinks it'd be a slightly nicer UX! 🤗💫

Reply:
Oh, and by the way, how do we feel about lettin' users ✨ add a pipeline ✨ from a path/to/pipeline.py 🛤️🐍, like with the pipeline register command? 🤔💭🎀